README.md

```diff
@@ -46,8 +46,7 @@ This scanner module provides the following API:
   }
   ```
 
-- `PUT /config/election` configures the scanner with an `election.json` as the
-  body
+- `PATCH /config` configures `election` with an `election.json` or `testMode`
```

Comment:
LOL PATCH. I guess I'm not the REST'iest of them all.

Reply:
oh I see ... this is a sneaky way to begin to add a small hint of graphql-like semantics... ok ok, respect.

Reply:
Maybe. I was mostly just following some stuff @jonahkagan did in Arlo-land.

Reply:
Nah I don't have any `PATCH` requests - opted for a nested path to update just one field of a resource